src/component/Layout/DefaultLayout.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import PropTypes from 'prop-types';
+
+function DefaultLayout({ children }) {
+  return (
+    <section>
+
+      <header>
+        <nav>
+          <ul>
+            <li><Link to="/books">Books</Link></li>
+            <li><Link to="/categories">Categories</Link></li>
```

Comment:
- [ ] Following the project requirements, in the navbar, the `<Link>` books should lead to the booklist location, and the `<Link>` categories should lead to the categories location. Please make sure that when the user clicks on the books link, only the books component is displayed, and that when the user clicks on the categories link, only the categories component is displayed.